simulariumio/cytosim/cytosim_converter.py

```diff
@@ -43,18 +43,21 @@ def _ignore_line(self, line: str) -> bool:
         """
         return len(line) < 1 or line[0:7] == "warning" or "report" in line
 
-    def _parse_fiber_data_dimensions(
-        self, fibers_lines: List[str]
+    def _parse_object_type_dimensions(
+        self,
+        data_lines: List[str],
+        is_fiber: bool,
     ) -> Tuple[List[int], int]:
         """
-        Parse a Cytosim fiber_points.txt output file to get the number
+        Parse a Cytosim output file containing objects
+        (fibers, solids, singles, or couples) to get the number
         of subpoints per agent per timestep
         """
         result = []
         t = -1
         s = 0
```

Comment:
these are indices so I did this for brevity, e.g. `result.n_subpoints[t][n_other_agents + n] = s + 1` would otherwise be `result.n_subpoints[time][n_other_agents + agent] = subpoint + 1` over and over. I guess I could change it but would rather not change it for everything right now